DiscordChatExporter.Cli/Commands/GetChannelsCommand.cs

```diff
@@ -36,6 +36,15 @@ public override async ValueTask ExecuteAsync(IConsole console)
             .ThenBy(c => c.Name)
             .ToArray();
 
+        var threads = Array.Empty<ChannelThread>();
+
+        if (IncludeThreads)
+        {
+            threads = (await Discord.GetGuildThreadsAsync(GuildId, cancellationToken))
+                .OrderBy(c => c.Name)
+                .ToArray();
+        }
```

Comment:
Can avoid mutation here:

```suggestion
        var threads = IncludeThreads
            ? (await Discord.GetGuildThreadsAsync(GuildId, cancellationToken))
                .OrderBy(c => c.Name)
                .ToArray()
            : Array.Empty<ChannelThread>();
```